cypress/e2e/Filters/NoTrials/NoTrials.feature

```diff
@@ -0,0 +1,66 @@
+Feature: No trials found behavior with filters
+	Scenario: User enters a combination of filters that returns no trials for diseases
+		Given "trialListingPageType" is set to "Disease"
+		And "dynamicListingPatterns" object is set to "Disease"
+		And the user navigates to "/lung-cancer?cfg=0&pn=1"
+		And user has typed 2 in the Age text input
+		And user has typed "30309" in the Location by ZIP Code text input
+		And user has selected 20 miles for the Radius dropdown
+		When user clicks the "Apply Filters" button
+		Then the page URL stays the same with the additional query parameter
+		And intro text displays
+		And Trials 0 of 0 displays below the intro text
```

Comment:
I would recommend to pass it as string parameter
`And "Trials 0 of 0" displays below the intro text`, and this will make step implementation dynamic 

```
Then('{string} displays below the intro text', (text) => {
	cy.get('.nci-spinner').should('not.exist', { timeout: 10000 });
	cy.get('.ctla-results__count').should('include.text', text);
});
```